server/src/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
I think it might be a better idea if we used the built-in body-parser in express
```js
express.json()
express.urlendcoded({ extended: false })
```
so we avoid installing a 3rd party library
Does this make sense?

Reply:
true
let's see if we get feedback on this